deploy-all.js

```diff
@@ -122,23 +122,37 @@ async function main() {
     const mockAddress = '0x00360d2b7d240ec0643b6d819ba81a09e40e5bcd';
 
     // register assets in datafeed
-    tokenInfo[networkName].forEach(async (token) => {
-      console.log(`Registering ${token.name}`);
+
+
+    const assetsToRegister = [
+      'ANT-T', 'BNT-T', 'BAT-T', 'BTC-T', 'DGD-T', 'DOGE-T', 'ETC-T', 'ETH-T', 'EUR-T',
+      'GNO-T', 'GNT-T', 'ICN-T', 'LTC-T', 'REP-T', 'XRP-T', 'SNGLS-T', 'SNT-T', 'MLN-T',
+    ];
+
+    for(const assetSymbol of assetsToRegister) {
+      console.log(`Registering ${assetSymbol}`);
+      const thisToken = tokenInfo[networkName].filter(token => token.symbol === assetSymbol)[0];
       await datafeed.methods.register(
-        token.address,
-        token.name,
-        token.symbol,
-        token.decimals,
-        token.url,
+        thisToken.address,
```

Comment:
liked `token` better then `thisToken` - as [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) as a keyword in js